diff --git a/imx/meta-bsp/recipes-graphics/mesa/mesa-demos_%.bbappend b/imx/meta-bsp/recipes-graphics/mesa/mesa-demos_%.bbappend
index 7dbf0ae..7b371c4 100644
--- a/imx/meta-bsp/recipes-graphics/mesa/mesa-demos_%.bbappend
+++ b/imx/meta-bsp/recipes-graphics/mesa/mesa-demos_%.bbappend
@@ -8,10 +8,7 @@
 DEPENDS = "mesa"
 
 REMOVE_GLU = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'glu x11', '', d)}"
-PACKAGECONFIG_remove_mx6q = "${REMOVE_GLU}"
-PACKAGECONFIG_remove_mx6dl = "${REMOVE_GLU}"
-PACKAGECONFIG_remove_mx6sx = "${REMOVE_GLU}"
-PACKAGECONFIG_remove_mx6sl = "${REMOVE_GLU}"
+PACKAGECONFIG_remove_mx6 = "${REMOVE_GLU}"
 
 PACKAGECONFIG_append_mx6 = " ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland vg', '', d)}"
 
